assets/javascripts/kitten/karl/pages/pre-deposit/components/carousel-section.js

```diff
@@ -0,0 +1,221 @@
+      </div>
+    )
+  }
+
+  renderTitle() {
+    const { viewportIsMOrLess } = this.props
+
+    return (
+      <Container>
+        <Grid>
+          <GridCol
+            col-l="5"
+            offset-l="1"
+            col-s="7"
+            col-xs="10"
+            style={{
+              ...styles.flex,
+              ...styles.spaceBetween,
+            }}
+          >
+            <Marger bottom="3" style={{ ...styles.flex, ...styles.title }}>
+              {!viewportIsMOrLess && (
+                <HorizontalStroke size="default" style={styles.stroke} />
+              )}
+              <Title
+                tag="h2"
+                modifier="quaternary"
+                margin={false}
+                className="has-overrided"
+              >
+                Nos créatrices et créateurs racontent leur campagne
+              </Title>
+            </Marger>
+          </GridCol>
+        </Grid>
+      </Container>
+    )
+  }
+
+  renderCarousel() {
+    const creators = [
+      {
+        imageProps: {
+          src: 'https://placehold.it/350x195/caf4fe/caf4fe',
+        },
+        title:
+          '« Le crowdfunding était un bon moyen de tester le projet face au public. »',
+        paragraph: 'Le Barboteur, péniche culturelle itinérante',
+      },
+      {
+        imageProps: {
+          src: 'https://placehold.it/350x195/caf4fe/caf4fe',
+        },
+        title:
+          '« Cette campagne était aussi une manière de montrer que notre crédibilité. »',
+        paragraph:
+          'Wheeliz, premier site de location de voitures aménagées pour les personnes en fauteuil entre particuliers',
+      },
+      {
+        imageProps: {
+          src: 'https://placehold.it/350x195/caf4fe/caf4fe',
+        },
+        title: '« Notre volonté était d’auto-éditer notre guide. »',
+        paragraph: 'Chilowé, guide des micro-aventures en île-de-France',
+      },
+      {
+        imageProps: {
+          src: 'https://placehold.it/350x195/caf4fe/caf4fe',
+        },
+        title: '« Notre volonté était d’auto-éditer notre guide. »',
+        paragraph: 'Chilowé, guide des micro-aventures en île-de-France',
+      },
+    ]
+
+    return (
+      <Container
+        fullWidthBelowScreenSize={SCREEN_SIZE_M}
+        style={{ position: 'relative' }}
+      >
+        <Carousel
+          itemMinWidth={220}
+          baseItemMarginBetween={20}
+          data={creators}
+          renderItem={({ item }) => (
+            <SimpleCard
+              title={item.title}
+              paragraph={item.paragraph}
+              tag="a"
+              href="#"
+              target="_blank"
+              rel="external"
+              imageProps={{
+                src: item.imageProps.src,
+                alt: '',
+              }}
+            />
+          )}
+        />
+      </Container>
+    )
+  }
+}
+
+const styles = {
+  background: {
+    position: 'relative',
+    overflow: 'hidden',
+    padding: 'pxToRem(100 0)',
```

Comment:
Ça ne peut pas fonctionner comme ça. Il faut faire :
```suggestion
    padding: `${pxToRem(100)} ${pxToRem(0)}`,
```